tox.ini

```diff
@@ -16,6 +17,7 @@ deps=
   pytest-asyncio
 
 commands=
+  python setup.py develop
```

Comment:
If you want to use this, please use [extras parameter](http://tox.readthedocs.io/en/latest/config.html#confval-extras=MULTI-LINE-LIST)